### PR TITLE
[Doc] PQ and DLQ does not support NFS

### PR DESCRIPTION
**Pull request by kaisecheng**
- adds Tips to PQ and DLQ to suggest not to use NFS as storage
- remove `queue.max_bytes` part as size checking for multiple pipelines is released
Fixed: #12097

### CODE REVIEW

File: docs/static/persistent-queues.asciidoc [hunk 1]
```diff
@@ -29,6 +29,7 @@ Persistent queues do not solve these problems:
 * A persistent queue does not handle permanent machine failures such as disk corruption, disk failure, and machine loss. 
 The data persisted to disk is not replicated.
 
+TIP: Use local filesystem for data integrity and performance. NFS is not supported.
```
Comment (karenzone) [inline docs/static/persistent-queues.asciidoc:32]:
```suggestion
TIP: Use the local filesystem for data integrity and performance. Network File System (NFS) is not supported.
```

**Review by karenzone — APPROVED** [review]
Left a suggestion inline for your consideration. Otherwise, LGTM

**Comment by kaisecheng** [comment]
@logstashmachine backport 7.17

**Comment by kaisecheng** [comment]
@logstashmachine backport 8.2

**Pull request merged**